addons/point_of_sale/static/src/scss/pos.scss

```diff
@@ -161,3 +161,18 @@ button {
 .pos .button.validation.with-more-button {
     height: calc(var(--btn-height-size) * 2);
 }
+
+// Set all the colors but the "no-color" one
```

Comment:
can you put a comment here explaining that these are just the colors from web, but with lower opacity? In the future it would be nice to have a way to do this w\o duplicating code